### ISSUE TITLE
Internship Badge

### ISSUE THREAD

**Issue by kbwagner** [openopps/openopps-platform#3001]
Who: Unpaid internship participants
What: Receive a badge
Why: as a visual indication of completion of the internship

Acceptance Criteria:
- When a hiring manager clicks the "Internship complete" link under the student's name and then the "Close internship" button, the selectee/alternate will receive an Internship badge.
- When the intern has completed two internships, the badge will appear with a 2 in the lower right corner
- When the intern has completed three internships, the badge will appear with a 3 in the lower right corner.
- After 3  internships, no additional internship badges will display
- The traditional newcomer badge will not be given to interns.  However, if the intern becomes a federal employee and then applies for a traditional opportunity, they will then receive a newcomer badge.

Badge Image for Interns: 
Final badges in .svg
[badges_intern.zip](https://github.com/openopps/OpenOppsTasks/files/2997054/badges_intern.zip)

Badges: https://github.com/openopps/openopps-platform/issues/3000

Future Note: The badge will be the same whether the internship is unpaid of VSFS.

**Comment by Kibrahacha67** [comment]
@kbwagner - Question, are these to display on the home page in a new block?  as the page now does  not support badges...

**Comment by kbwagner** [comment]
@Kibrahacha67 that is a separate ticket 3619.  This ticket creates the badges and puts them in the system for use and the other ticket puts it on the user's profile.

**Comment by Kibrahacha67** [comment]
@kbwagner ok so there will be no way to see them until that ticket is completed, so dependency for this ticket...and this one will be on hold till that one is completed.

**Comment by kbwagner** [comment]
@Kibrahacha67 yes that makes sense.  Most of the close out tickets are going to be somewhat dependent on each other I think.

**Comment by Kibrahacha67** [comment]
@kbwagner all good I just have to keep track of which ones first so testing doesn't get hemmed up...  :)

**Comment by Kibrahacha67** [comment]
8-6-19 LHammond in testing
Testing completed moving to review.

**Comment by Piglady64** [comment]
@kbwagner One of my students did get the correct badge for 1 internship. I don't have one with 2 or 3.
![image](https://user-images.githubusercontent.com/15252970/64462250-ca27f480-d0cd-11e9-8bee-834b2d411cea.png)

**Comment by kbwagner** [comment]
@Piglady64 so oddly Penny has one with a 2.  Has she applied to any other internships and been selected?

**Comment by Piglady64** [comment]
This may be a bug. Penny only has one "completed" internship.
![image](https://user-images.githubusercontent.com/15252970/64538977-9f1aec00-d2eb-11e9-95c2-0590500f3596.png)

**Comment by Piglady64** [comment]
my student "penny coin" only has 2 internship "completed" but has a badge with a 3. returning to development
![image](https://user-images.githubusercontent.com/15252970/64557615-edda7d00-d30f-11e9-86d4-c83ac8a22c54.png)

![image](https://user-images.githubusercontent.com/15252970/64557661-0b0f4b80-d310-11e9-9045-4139033b8fe4.png)

**Comment by raywilliams** [comment]
@kbwagner @Piglady64 Can this be retested with a new student account and new cycles?

**Comment by kbwagner** [comment]
@raywilliams yes we can retest - but probably not until next week

**Comment by kbwagner** [comment]
Ready for final review
